include/pybind11/pytypes.h

```diff
@@ -1738,35 +1894,36 @@ class memoryview : public object {
 
         See also: Python C API documentation for `PyMemoryView_FromBuffer`_.
 
-        .. _PyMemoryView_FromMemory: https://docs.python.org/c-api/memoryview.html#c.PyMemoryView_FromMemory
-     \endrst */
```

Comment:
clang-format still seems to be unaware of reStructuredText.

Previously the doc generation would fail after I ran clang-format (clang 12).

Are we just getting lucky now?